packages/UsageAnalysis/files/Test Track/Viewers/3d Scatter plot.md

```diff
@@ -0,0 +1,15 @@
+### 3d Scatter plot
+
+1. Open the **demog dataset**.
```

Comment:
Link the dataset to test case

Reply:
added "datasets": ["System:DemoFiles/demog.csv"]